### PR TITLE
itzliam domain

### PR DESCRIPTION
**Pull request by LiamGaming23**
To make our job easier, please spend time to review your application before submitting. 

#### Add a x between [] if you meet that requirement example `[x]`
- [x] You're not using Vercel or Netlify.
- [x] You have completed your website, there's no type of placeholder at the website. **This requirement can be raised if the domain you're registering is for emails**
  - Link to website: 
- [x] The website is reachable.
- [x] The CNAME record doesn't contain any slash.
- [ ] There's enough information at the `owner` field.
   - You should have your email presented. If you don't want to share email, you can leave email an empty string (`""`) and add any other social such as Discord/Twitter/etc.
 
 #### Feel free to join our discord server for more updates or any help related to the service :D - https://discord.gg/PZCGHz4RhQ

### CODE REVIEW

File: domains/itzliam [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+"owner": {
+"username": "liamgaming23",
+"email": "liamstickman123@gmail.com"
+},
+"record": {
+"CNAME": "liamgaming23.github.io/liam.github.io"
```
Comment (WilliamDavidHarrison) [inline domains/itzliam:7]: Please remove the `/` and everything after it.

Comment (LiamGaming23) [reply domains/itzliam]: did it

**Review by wdhdev — CHANGES_REQUESTED** [review]
Please rename your file to: Itzliam.json

**Comment by LiamGaming23** [comment]
> 

done

**Review by wdhdev — APPROVED** [review]

**Review by andrewstech — APPROVED** [review]

**Pull request merged**